go/cli/mcap/README.md

```diff
@@ -32,12 +32,14 @@ To install using [Homebrew](https://brew.sh) on macOS or Linux, run:
 
     brew install mcap
 
-#### Using go
+#### Using go (not recommended)
```

Comment:
this feels unnecessary - users can read the warning and make their own judgement around what to do.

Reply:
It was to address @jtbandes concern. But it does feel a little in your face.